MSAL/src/MSALResult.m

```diff
@@ -135,10 +135,6 @@ + (MSALResult *)resultWithMSIDTokenResult:(MSIDTokenResult *)tokenResult
     }
     
     NSString *accessToken = [authScheme getClientAccessToken:tokenResult.accessToken popManager:popManager error:error];
-    if (!accessToken)
```

Comment:
Could you also update accessToken property in the MSALResult file to be nullable?

It's currently non null here:

```
@interface MSALResult : NSObject

#pragma mark - Token response

/** The Access Token requested. */
@property (readonly, nonnull) NSString *accessToken;
```

Reply:
Thanks for the review. In addition to the properties you pointed out, I've changed the properties that depend on `accessToken` to `nullable`

Reply:
Thanks!